patches/server/0005-Paper-config-files.patch

```diff
@@ -465,7 +464,8 @@ index 0000000000000000000000000000000000000000..52b02cb1f02d1c65b840f38cfc8baee5
 +
 +@SuppressWarnings({"CanBeFinal", "FieldCanBeLocal", "FieldMayBeFinal", "NotNullFieldNotInitialized", "InnerClassMayBeStatic"})
 +public class GlobalConfiguration extends ConfigurationPart {
-+    static final int CURRENT_VERSION = 28;
++    private static final Logger LOGGER = LogUtils.getLogger();
++    static final int CURRENT_VERSION = 28; // (when you change the version, change the comment, so it conflicts on rebases): <insert changes here>
```

Comment:
huh?

Reply:
This is so if you have 2 PRs that make configuration changes, and increment the version, there's less chance of the second PR being merged without noticing that the version was already incremented.